docs/components/modeler/web-modeler/process-application-pipeline.md

```diff
@@ -0,0 +1,92 @@
+---
+id: process-application-pipeline
+title: Process application development pipeline
+description: The Web Modeler provides a lightweight development pipeline you can use to quickly develop and progress low-risk process application releases through the stages of a standard development life cycle.
+---
+
+import DeployProcessApplicationDiagramImg from './img/process-applications/diagram-process-application-pipeline.png'
+import DeployStagesImg from './img/process-applications/define-stages.png'
+
+You can use the Web Modeler development pipeline to quickly develop and progress low-risk process application releases through the stages of a standard development life cycle.
```

Comment:
```suggestion
You can use the Web Modeler to quickly develop and progress low-risk process application releases through the stages of a standard development lifecycle.
```

Reply:
Here's one way to clean up the language -please check through the PR as I didn't comment on all instances of development lifecycle / pipeline